src/js/main.js

```diff
@@ -32,9 +31,8 @@ export function init(el, context, config) {
             hiddenDesc.classList.toggle('expanded');
         };
 
-        const emailIframe = builder.querySelector('.js-email-sub__iframe');
-
         builder.querySelector('.docs__poster--loader').addEventListener('click', function() {
+            const player = new PimpedYouTubePlayer(youTubeId, builder, '100%', '100%', config);
```

Comment:
I think perhaps the version deployed had diverged from the version in git, so it is hard to reason about how it was working before.